Source/WebCore/platform/calc/CalcExpressionOperation.cpp

```diff
@@ -132,6 +133,14 @@ float CalcExpressionOperation::evaluate(float maxValue) const
     case CalcOperator::Tan: {
         if (m_children.size() != 1)
             return std::numeric_limits<double>::quiet_NaN();
+        double x = std::fmod(m_children[0]->evaluate(maxValue), piDouble * 2);
```

Comment:
We should share this code, not write it out twice.

Reply:
This whole file is duplicated unfortunately, we've had a bug where the `hypot()` function was wrong in only one of the two files, so this is something I'd like to address. I have some ideas how to refactor to prevent duplication altogether, but I'd like to do it separately to catch potential regressions.